ydb/core/blobstorage/dsproxy/dsproxy_get_block.cpp

```diff
@@ -0,0 +1,139 @@
+#include "dsproxy.h"
+#include "dsproxy_mon.h"
+#include "dsproxy_quorum_tracker.h"
+#include <ydb/core/blobstorage/base/blobstorage_events.h>
+#include <ydb/core/blobstorage/vdisk/common/vdisk_events.h>
+
+namespace NKikimr {
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// GET BLOCK request
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+class TBlobStorageGroupGetBlockRequest : public TBlobStorageGroupRequestActor {
+    const ui64 TabletId;
+    ui64 Generation;
+    const TInstant Deadline;
+    ui64 Requests;
+    ui64 Responses;
+    TGroupQuorumTracker QuorumTracker;
+
+    void Handle(TEvBlobStorage::TEvVGetBlockResult::TPtr &ev) {
+        ProcessReplyFromQueue(ev->Get());
+        const NKikimrBlobStorage::TEvVGetBlockResult &record = ev->Get()->Record;
+        Y_ABORT_UNLESS(record.HasStatus());
+        const NKikimrProto::EReplyStatus status = record.GetStatus();
+        Y_ABORT_UNLESS(record.HasVDiskID());
+        const TVDiskID vdisk = VDiskIDFromVDiskID(record.GetVDiskID());
+
+        DSP_LOG_LOG_S(PriorityForStatusInbound(status), "DSPGB01", "Handle TEvVGetBlockResult"
+            << " status# " << NKikimrProto::EReplyStatus_Name(status).data()
+            << " From# " << vdisk.ToString()
+            << " NodeId# " << Info->GetActorId(vdisk).NodeId());
+
+        if (record.HasGeneration()) {
+            const ui64 generation = record.GetGeneration();
```

Comment:
generations are 32-bit, why ui64? and why additional variable, why not write in in single line?